css/style.css

```diff
@@ -0,0 +1,243 @@
+/*
+The CSS code gets to executed from top to bottom and left ro right
+*/
+
+
+/* CSS reset
+(all the browsers will see the website in the same by setting every tag to 0 (the website will look the same on all browsers))
+*/
+
+* {
+    margin: 0;
+    padding: 0;
+    border: 0;
+    font-size: 100%;
+    /*The box-sizing property is used to tell the browser what the sizing properties (width and height) should include border-box.*/
+    box-sizing: border-box;
+    /*reset Chrome user agent styles*/
+        -webkit-margin-before: 0px;
+        -webkit-margin-after: 0px;
+        -webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
+    -webkit-padding-before: 0px;
+    -webkit-padding-after: 0px;
+    -webkit-padding-start: 0px;
+    -webkit-padding-end: 0px;
+}
+
+
+/* General styles
+(the styles used everywhere in the page (like the Body) or styles which are used in more than 2 of the modules bellow (h2, h3, h4, h5, h6, p, a, ul, li))
+*/
+
+body {
+    font-size: 16px;
+    font-family: 'Special Elite', cursive;
+    background-image: url(../images/camera-min.jpg);
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+ul {
+    list-style-type: none;
+}
+
+button {
+    text-transform: uppercase;
+    border-radius: 4px;
+    color: white;
+    background-color: darkorange;
+}
+
+.startQuiz {
+    
```

Comment:
Be consistent. If you put 1 space or 2 spaces between blocks, it should be consistent throughout. Generally there shouldn't be any space inside the rule block.